reference/compiling_for_android.rst

```diff
@@ -215,14 +198,14 @@ failure should be presented there.
 
 Seek assistance if you can't figure it out.
 
-Application exits immediately
+2.) Application exits immediately
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 If the application runs but exits immediately, there might be one of the
 following reasons:
 
--  libgodot_android.so is not in ``libs/armeabi``
+-  libgodot_android.so is not in ``lib/armeabi-v7a``
```

Comment:
There is no bug in Windows buildsystem. Unix is the same as Windows one.
Both Windows and Unix has libs/{release,debug}/armeabi-v7a

The phrase below :
If the application runs but exits immediately, there might be one of the following reasons:
libgodot_android.so is not in ``lib/armeabi-v7a``

Doesn't refer to Gradle based Godot Android project, instead it refers to Exported Apk(s), in this case android_debug.apk and android_release.apk

Gradle based Godot Android project VS Exported Apks
libs/{release,debug}/armeabi-v7a  -- VS --  lib/armeabi-v7a

:-)